### PR TITLE
Rectify error arises when syncdb command is run

### PR DESCRIPTION
**Pull request by aseemmittal**

### CODE REVIEW

File: src/librehatti/suspense/forms.py [hunk 1]
```diff
@@ -154,12 +154,11 @@ class Meta:
         widgets = {'session': forms.HiddenInput(),\
         'voucher_no': forms.HiddenInput()}
 
-    choices=[ (o.id, str(o)) for o in FinancialSession.objects.all()]
-    receipt_session = forms.ChoiceField(choices)
-
```
Comment (jasvir99) [inline src/librehatti/suspense/forms.py]: What was the error and how this section contributed to that?

File: src/librehatti/suspense/forms.py [hunk 2]
```diff
@@ -154,12 +154,11 @@ class Meta:
...
     def __init__(self, *args, **kwargs):
         super(CarTaxiAdvance_form, self).__init__(*args, **kwargs)
         self.fields['spent'].widget.attrs={'class':'form-control'}
         self.fields['advance'].widget.attrs={'class':'form-control'}
         self.fields['receipt_no'].widget.attrs={'class':'form-control'}
-        self.fields['receipt_session'].widget.attrs={'class':'form-control'}
```
Comment (shagun30) [inline src/librehatti/suspense/forms.py]: Difference between this line and the next line.

Comment (jasvir99) [inline src/librehatti/suspense/forms.py]: @shagun30, I think that's an EOL

File: src/librehatti/suspense/forms.py (hunk 1 above)
Comment (aseemmittal) [inline src/librehatti/suspense/forms.py]: The error was "django.db.utils.ProgrammingError: (1146," Table 'librehatti.voucher_financialsession' doesn't exist") "
I think whenever we try to migrate database,  django tries to find financial session table for a form field which django wasn't able to find.

**Pull request merged**